go/api/base_client.go

```diff
@@ -2954,6 +2954,136 @@ func (client *baseClient) BLMPopCount(
 	return handleStringToStringArrayMapOrNilResponse(result)
 }
 
+// Blocks the connection until it pops and returns a member-score pair from the first non-empty sorted set, with the
+// given keys being checked in the order they are provided.
+// BZMPop is the blocking variant of [api.ZMPop].
+//
+// Note:
+//   - When in cluster mode, all keys must map to the same hash slot.
+//   - BZMPop is a client blocking command, see [Blocking Commands] for more details and best practices.
+//
+// Since:
+//
+//	Valkey 7.0 and above.
+//
+// See [valkey.io] for details.
+//
+// Parameters:
+//
+//	keys          - An array of keys to lists.
+//	scoreFilter - The element pop criteria - either MIN or MAX to pop members with the lowest/highest scores accordingly.
+//	timeoutSecs   - The number of seconds to wait for a blocking operation to complete. A value of 0 will block indefinitely.
+//
+// Return value:
+//
+//	An object containing the following elements:
+//	- The key name of the set from which the element was popped
+//	- An array of member scores of the popped elements in ascending order.
+//	Returns nil if no member could be popped and the timeout expired.
+//
+// For example:
+//
+//	result, err := client.ZAdd("my_list", map[string]float64{"five": 5.0, "six": 6.0})
+//	result, err := client.BZMPop([]string{"my_list"}, api.MAX, float64(0.1))
+//	result["my_list"] = []MemberAndScore{{Member: "six", Score: 6.0}}
+//
+// [valkey.io]: https://valkey.io/commands/bzmpop/
+// [Blocking Commands]: https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#blocking-commands
+func (client *baseClient) BZMPop(
```

Comment:
It is `BaseClient` already, no?
```suggestion
func (client *BaseClient) BZMPop(
```

Reply:
No, for docs we changed `glideClient` to `GlideClient` instead to prevent having to expose `baseClient`. This still allows `pkgsite` to document `BZMPop`